src/sdl_utils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "sdl_gl.h"
+#include <iostream>
+
+void sdl_application_abort(const char * msg)
+{
+    std::cout << SDL_GetError() << std::endl;
+    SDL_Quit();
+    exit (-1);
+}
+
+void getInfo(void)
+{
+    SDL_version compiled;
+    SDL_version linked;
+    SDL_VERSION(&compiled);
+    SDL_GetVersion(&linked);
+
+    fprintf(stdout, "We compiled against SDL version %d.%d.%d ...\n",compiled.major, compiled.minor, compiled.patch);
+    fprintf(stdout, "And we are linking against SDL version %d.%d.%d.\n", linked.major, linked.minor, linked.patch);
+
+    int glMajor = 0;
+    int glminor = 0;
+    glGetIntegerv(GL_MAJOR_VERSION, &glMajor);
+    glGetIntegerv(GL_MINOR_VERSION, &glminor);
+    std::cout << "Current OpenGL version : " << glMajor << "." << glminor << std::endl;
+}
+
+void getTextureInfo( SDL_Texture * texture)
+{
+    int loop_value = 100;
+    unsigned int * aTextureInfo_format = 0;
+    int * aTextureInfo_access = 0;
+    int * aTextureInfo_w = 0;
+    int * aTextureInfo_h = 0;
+    int anErr = 0;
+    int count = 0;
+
+    if ( count % loop_value == 0 )

```

Comment:
Copy and paste error or left over debug? Please cleanup the unused looping logic, this isn't looping.

There appears to be no point to any of this "count" or "loop_value" stuff.
Every time this function is called, it'll initialize the variables (they aren't static), so the (0 % 100 == 0) is always true.